contracts/src/v0.8/llo-feeds/v0.4.0/DestinationVerifier.sol

```diff
@@ -323,19 +323,22 @@ contract DestinationVerifier is IDestinationVerifier, ConfirmedOwner, TypeAndVer
         return timestamp;
     }
 
-    function findActiveConfig(uint256 timestamp) internal view returns (DONConfig memory){
+   function findActiveConfig(uint256 timestamp) internal view returns (DONConfig memory){
         DONConfig memory activeDONConfig;
 
         // 99% of the time the signer config will be the last index, however for historic reports generated by a previous configuration we'll need to cycle back
-        for(uint256 i = s_DONConfigs.length - 1; i >= 0; --i) {
-            if(s_DONConfigs[i].activationTime < timestamp) {
+       uint256 i = s_DONConfigs.length;
+       while (i > 0) {
```

Comment:
@Fletch153  turned this into a while loop to avoid the underflow, and only one decrement operation.